relay-server/src/envelope.rs

```diff
@@ -881,6 +893,18 @@ impl Envelope {
         self.headers.other.insert(name.into(), value.into())
     }
 
+    /// Returns the internal early enforcement header.
+    ///
+    /// See [`EnvelopeHeaders::early_enforcement`].
+    pub fn get_early_enforcement(&self) -> &Enforcement {
```

Comment:
nit

```suggestion
    pub fn early_enforcement(&self) -> &Enforcement {
```

https://doc.rust-lang.org/1.0.0/style/style/naming/README.html#getter/setter-methods-[rfc-344]

Reply:
TIL